internal/lib/lib.go

```diff
@@ -96,10 +96,9 @@ func SetGRBACSupport(val string) {
 	controllerVersion := utils.CtrlVersion
 	if gRBAC && CheckControllerVersionCompatibility(controllerVersion, "<", ControllerVersion2015) {
 		// GRBAC is supported from 20.1.5 and above
-		utils.AviLog.Infof("Disabling GRBAC as controller version is less than %v", ControllerVersion2015)
+		utils.AviLog.Infof("Disabling GRBAC as current controller version %s is less than %s.", ControllerVersion2015)
```

Comment:
Nit: Missed "controllerversion" parameter while printing msg.

Reply:
Another good one - I am taking back the review till this is fixed.

Reply:
ah thanks will do